modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -1134,7 +1134,7 @@ translationTest(
 		});
 
 		if (await textBox.isHidden()) {
```

Comment:
I'm not sure if we can apply the new util [playwright/utils/openFieldset.ts](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/openFieldset.ts) here.

Please, can you check?